src/NuGet.Core/NuGet.Protocol/HttpSource/HttpRetryHandlerRequest.cs

```diff
@@ -22,8 +22,14 @@ public HttpRetryHandlerRequest(HttpClient httpClient, Func<HttpRequestMessage> r
             RequestFactory = requestFactory;
             CompletionOption = HttpCompletionOption.ResponseHeadersRead;
             MaxTries = DefaultMaxTries;
+
+#if DEBUG
+            RequestTimeout = TimeSpan.FromSeconds(300);
```

Comment:
This feels pretty arbitrary to me. :/ 
I get that it's for debugging purposes but debugging is not just loading the PM UI. 
I'd vote to revert it.

Reply:
It's 5 minutes. You can't do any useful debugging in this area in less time than that. Can be reverted, but I and others will have to add it back every time the UI is debugged.

Reply:
People didn't chekc it in before. 
it's very much a function of how long you are debugging, and that depends on what you are debugging specifically.

Reply:
Removing